tests/unit/macaroons/test_models.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from warehouse.macaroons import models
+
+
+def test_generate_key():
+    key = models._generate_key()
```

Comment:
Is it worth checking that generating keys twice lead to 2 different keys ?

Reply:
`_generate_key` is a straight-line call to `os.urandom`, so I don't think this test would add much (at most it would tell us that `os.urandom` on whatever machine is running the tests isn't really random, but that isn't likely to be a production machine).